models/DataObject/AbstractObject.php

```diff
@@ -1062,7 +1062,7 @@ public function getUserModification()
     }
 
     /**
-     * @param int $o_id
+     * @param int|null $o_id
```

Comment:
See https://github.com/pimcore/pimcore/blob/89ecd1b81d4e955164027ca3a9129fa4b0d82679/models/DataObject/Service.php#L215

In https://github.com/pimcore/pimcore/pull/11826 I changed also the method body, so it will not cast to 0 on copy